src/main/java/io/reactivex/disposables/CompositeDisposable.java

```diff
@@ -223,7 +225,14 @@ void dispose(OpenHashSet<Disposable> set) {
         }
         if (errors != null) {
             if (errors.size() == 1) {
-                throw Exceptions.propagate(errors.get(0));
+                Throwable ex = errors.get(0);
+                if (ex instanceof Error) {
+                    throw (Error)ex;
+                }
+                if (ex instanceof RuntimeException) {
```

Comment:
Sounds good to me. These exception helper methods could _always_ return an unchecked exception forcing you to use `throw` and ensuring code coverage catches it.